sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/implementation/OpenAIClientImpl.java

```diff
@@ -2083,39 +1944,35 @@ public PollerFlux<BinaryData, BinaryData> beginBeginAzureBatchImageGenerationAsy
      *     }
      * }
      * }</pre>
-     *
+     * 
      * @param imageGenerationOptions Represents the request data used to generate images.
      * @param requestOptions The options to configure the HTTP request before HTTP client sends it.
      * @throws HttpResponseException thrown if the request is rejected by server.
      * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
      * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
      * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
      * @return the {@link SyncPoller} for polling of a polling status update or final response payload for an image
-     *     operation.
+     * operation.
      */
     @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
-    public SyncPoller<BinaryData, BinaryData> beginBeginAzureBatchImageGeneration(
-            BinaryData imageGenerationOptions, RequestOptions requestOptions) {
-        return SyncPoller.createPoller(
-                Duration.ofSeconds(1),
-                () -> this.beginAzureBatchImageGenerationWithResponse(imageGenerationOptions, requestOptions),
-                new SyncDefaultPollingStrategy<>(
-                        new PollingStrategyOptions(this.getHttpPipeline())
-                                .setEndpoint("{endpoint}/openai".replace("{endpoint}", this.getEndpoint()))
-                                .setContext(
-                                        requestOptions != null && requestOptions.getContext() != null
-                                                ? requestOptions.getContext()
-                                                : Context.NONE)
-                                .setServiceVersion(this.getServiceVersion().getVersion())),
-                TypeReference.createInstance(BinaryData.class),
-                TypeReference.createInstance(BinaryData.class));
+    public SyncPoller<BatchImageGenerationOperationResponse, BatchImageGenerationOperationResponse>
```

Comment:
Same here as above. The types for the `SyncPoller` changed from `<BinaryData, BinaryData>` to `<BatchImageGenerationOperationResponse, BatchImageGenerationOperationResponse>`